redi-connect-front/src/pages/front/login/Login.tsx

```diff
@@ -78,107 +68,90 @@ export default function Login() {
+            className="title--border"
+          >
+            Sign-in
+          </Heading>
+          <Heading size={4} renderAs="p" subtitle>
+            Enter your email and password below.
+          </Heading>
+          <form onSubmit={e => e.preventDefault()} className="login-form">
+            <Form.Field>
+              <Form.Control>
+                <Form.Input
+                  id="username"
+                  name="username"
+                  type="email"
+                  color={
+                    formik.touched.username && Boolean(formik.errors.username)
+                      ? "danger"
+                      : null
+                  }
+                  placeholder="Email"
+                  value={formik.values.username}
+                  onChange={onChange.bind(null, "username")}
+                  disabled={formik.isSubmitting}
+                />
+              </Form.Control>
+              {formik.touched.username && Boolean(formik.errors.username) && (
+                <Form.Help color="danger">This email is invalid</Form.Help>
+              )}
+            </Form.Field>
 
-                <Form.Field>
-                  <Form.Control>
-                    <Form.Input
-                      id="password"
-                      name="password"
-                      type="password"
-                      color={
-                        formik.touched.password &&
-                        Boolean(formik.errors.password)
-                          ? 'danger'
-                          : null
-                      }
-                      placeholder="Password*"
-                      value={formik.values.password}
-                      onChange={onChange.bind(null, 'password')}
-                      disabled={formik.isSubmitting}
-                    />
-                  </Form.Control>
-                  {loginError && (
-                    <Form.Help color="danger">{loginError}</Form.Help>
-                  )}
-                </Form.Field>
-                <Form.Field
-                  className="login-form__reset-password"
-                  textTransform="uppercase"
-                >
-                  <Link to="/front/reset-password/request-reset-password-email">
-                    Forgot your password?
-                  </Link>
-                </Form.Field>
-                <Form.Field className="login-form__submit">
-                  <Button
-                    onClick={formik.submitForm}
-                    type="submit"
-                    color="primary"
-                    variant="contained"
-                    fullWidth
-                    disabled={formik.dirty && formik.isValid ? false : true}
-                  >
-                    Log in
-                  </Button>
-                </Form.Field>
-              </form>
-            </Columns.Column>
-          </Columns>
-        </Section>
-      </Container>
-    </LoggedOutLayout>
+            <Form.Field>
```

Comment:
I see simple, testable components emerging here along the lines of DRY. I'd make the argument simple refactoring is worthwhile to already do, seeing how many input forms we'll be working with down the line.

What about an "atomic" component along these lines?

```
interface Props {
  name: String;
  type: 'email' | 'text' | 'password'; // and make 'text' the default one given null?
  color: String;
  placeholder: String;
  value: String;
  onChange: Function;
  disabled: Boolean;
}

function FormInput (props: Props) {
  return (
    <Form.Field>
      <Form.Control>
        <Form.Input
          /* blabla pass in all props */
        >
        </Form.Input>
      </Form.Control>
    </Form.Field>
  )
}
```